packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -21,28 +18,57 @@ const getLocation = (location_list, value, type) => {
     return '';
 };
 
+const SetCurrencyHeader = ({ has_target, has_real_account, has_currency, items, step }) => (
+    <>
+        {(!has_real_account || has_target) && (
+            <>
+                <DesktopWrapper>
+                    <FormProgress steps={items} current_step={step} />
+                </DesktopWrapper>
+                <MobileWrapper>
+                    <div className='account-wizard__header-steps'>
+                        <h4 className='account-wizard__header-steps-title'>
+                            <Localize
+                                i18n_default_text='Step {{step}}: {{step_title}} ({{step}} of {{steps}})'
+                                values={{
+                                    step: step + 1,
+                                    steps: items.length,
+                                    step_title: items[step].header.title,
+                                }}
+                            />
+                        </h4>
+                        {items[step].header.active_title && (
+                            <h4 className='account-wizard__header-steps-subtitle'>{items[step].header.active_title}</h4>
+                        )}
+                    </div>
+                </MobileWrapper>
+            </>
+        )}
+        <DesktopWrapper>
+            {has_real_account && !has_target && (
+                <div className='account-wizard__set-currency'>
+                    {!has_currency && (
+                        <p>
+                            <Localize i18n_default_text='You have an account that do not have currency assigned. Please choose a currency to trade with this account.' />
```

Comment:
This text does not seem grammatically correct.
> You have an account that does not have currency assigned.

As opposed to:
> You have an account that do not have currency assigned.

Can we get these checked again?

Reply:
All texts are under copy check right now, Will address all text in another card.